### ISSUE TITLE
filter on dataset name does not work

### ISSUE THREAD

**Issue by andersfi** [gbif-norway/helpdesk#190]
We use the term dataset name to register project data from NBICs "artsprosjekt" so that NBIC can verify that the data have been published according to the agreement. It is hence important that this workflow actually works. We have at NTNU-VM run into a bug here. 

The filtering on dataset name in the GBIF portal seems not to be working. For example, the occurrence https://www.gbif.org/occurrence/4948367391 have the dataset name "	Artsdatabanken Artsprosjekt_7-20_Rotifers - small coastal pounds from Agder, Trøms and Finnmark". Filtering for this on https://www.gbif.org/occurrence/search?publishing_org=a8144f37-5ff7-4137-9400-94b5b2ea4ec4&advanced=1&dataset_name=Artsdatabanken%20Artsprosjekt_7-20_Rotifers%20-%20small%20 does yield any hits on the portal. 

This error occurred after renaming and republishing the dataset.

**Comment by dagendresen** [comment]
When I search I get 528 hits

https://www.gbif.org/occurrence/search?publishing_org=a8144f37-5ff7-4137-9400-94b5b2ea4ec4&advanced=1&dataset_name=Artsdatabanken%20Artsprosjekt_7-20_Rotifers%20-%20small%20coastal%20pounds%20from%20Agder,%20Tr%C3%B8ms%20and%20Finnmark

![Screenshot 2024-10-14 at 09 14 40](https://github.com/user-attachments/assets/e1c34be2-079e-43f8-ad82-75b049f1e33b)

**Comment by aaltenburger2** [comment]
Related to the original question, I think the Provenance section, and therein the term project, would be a more natural place to include this information.
![image](https://github.com/user-attachments/assets/1f1628b4-b206-4467-b7e2-7f8598854b4e)
I can't find either the section or the term at https://dwc.tdwg.org/terms/. Do we have a field in MusIT that gets exported to the project field?

**Comment by dagendresen** [comment]
I believe that these terms are from EML (Ecological Metadata Language) -- mixed with some terms created by GBIF on dataset level only. And thus not available at record level. There has been a well documented request for projectID, projectName, project funder, etc for Darwin Core (and record-level documentation of project data) -- but somebody needs to make the effort to make the term request and maybe create a TDWG Task Group to develop such terms.

**Comment by aaltenburger2** [comment]
I am happy to contribute requesting those terms. Can you share what has been documented already?

**Comment by dagendresen** [comment]
We can try to collect some relevant references and GitHub issues together? I mean to recall that at least Sharon Grant (Field Museum) and Ming (AntaBIF) have been posting requests and suggestions. We can search for these on the GBIF and TDWG GitHub. I suggest searching GitHub repositories under GBIF and TDWG for search keywords "datasetID" and "datasetName".

* ProjectID issues GBIF Portal-Feedback: https://github.com/gbif/portal-feedback/issues?q=projectID
* ProjectIDs on individual records, rather than a dataset as a whole: https://github.com/gbif/pipelines/issues/836
* Make the metadata projectID field multivalue: https://github.com/gbif/ipt/issues/1927
* dwc:datasetName use to group datasets, yes, no, options please? https://github.com/tdwg/dwc-qa/issues/199
* Option to add several projects in the Project data metadata page: https://github.com/gbif/ipt/issues/1780
* Project Information - Darwin Core Hour Input Form 2/14/2017: https://github.com/tdwg/dwc-qa/issues/37

**Comment by aaltenburger2** [comment]
Thank you Dag! I see, it has been discussed previously, and a workaround was implemented by allowing the projectID metadata field to accept multiple values (https://github.com/gbif/pipelines/issues/836). However, this solution still does not support projectIDs on individual records. Instead of creating a new issue, should we reopen the above GitHub issue #836 and request that projectID be added to DwC at the record level? I anticipate this might create issues with the projectID field in the GBIF metadata. What I want/need/suggest are "project name," "projectID," "funder name," and "funder ID" as DwC terms at the record level.

**Comment by dagendresen** [comment]
Hi, I think that the appropriate chain of actions would be to (1) introduce project terms to Darwin Core (or another data standard) and then (2) introduce the TDWG terms to the GBIF application profiles we use with the IPT etc.  Sort of top-dowm, letting the data standards rule applications.

Jumping in at the GBIF issue 836 would mean minting temporary (?) project terms in the GBIF namespace that have not passed TDWG standardization. Sort of bottom-up letting practice rule the data standards... (if somebody makes the effort to integrate such practice into the data standards...)

Both paths are of course possible :-)

Then there is of course also the possibility to FIND the project terms in other non-TDWG data standards - and to promote using such standardized terms in the GBIF application profiles... Developing a new TDWG standard or addition to DwC would of course also involve exploring how other data standards describe projects!!!

**Comment by aaltenburger2** [comment]
I agree with your top down approach. I couldn't find a discussion about it on the TDWG github. Should I start one?

**Comment by dagendresen** [comment]
I suspect that there might be plenty already on various TDWG repositories... :-)
There are always the DwC-QnA (FAQ) threads to explore more and to post follow.up questions to...?
https://github.com/tdwg/dwc-qa/issues?q=is%3Aissue+is%3Aopen+project
https://github.com/tdwg/dwc-qa/issues/37
https://github.com/tdwg/dwc-qa/issues/83
https://github.com/tdwg/dwc-qa/issues/100
https://github.com/tdwg/dwc-qa/issues/199

**Comment by aaltenburger2** [comment]
https://github.com/tdwg/dwc/issues/527

**Comment by kkongshavn** [comment]
> Related to the original question, I think the Provenance section, and therein the term project, would be a more natural place to include this information. ![image](https://private-user-images.githubusercontent.com/96289489/376141296-1f1628b4-b206-4467-b7e2-7f8598854b4e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM0NzM2NTgsIm5iZiI6MTczMzQ3MzM1OCwicGF0aCI6Ii85NjI4OTQ4OS8zNzYxNDEyOTYtMWYxNjI4YjQtYjIwNi00NDY3LWI3ZTItN2Y4NTk4ODU0YjRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA2VDA4MjIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwMmM2YzQzYThkNWQxOWJmYTM4Y2Q1ODJjZjMwOWJkNjQ0YTRmYjNiM2MwOGQ5ZGMyNDc0MWIyNmQ4YWIxMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1s-qeHQul7BITCkjgb3pN2aihDCrm545frGj2bboe-o) I can't find either the section or the term at https://dwc.tdwg.org/terms/. Do we have a field in MusIT that gets exported to the project field?

Yes, there is a field in musit - it's the one for "Dataset" (which in GBIF besomes "DatasetName", I believe), that we use all the time - and that we use quite a lot to filter our results. So please make sure not to lose the ability to filter on this in GBIF, I was so happy when they implemented it!
